File: INICIANTE/1101/main.py
def formatted_print(m, n):
    if (m > n):
        m, n = n, m
    sequence = range(m, n + 1)
    print(f"{' '.join(map(str, sequence))} Sum={sum(sequence)}")


m, n = map(int, input().split())
while m > 0 and n > 0:
    formatted_print(m, n)
    m, n = map(int, input().split())

File: INICIANTE/1101/main.dart
import 'dart:io';

void formattedPrint(int m, int n) {
  if (m > n) [m, n] = [n, m];
  List<int> sequence = List<int>.generate(n - m + 1, (i) => i + m);
  print('${sequence.join(' ')} Sum=${sequence.reduce((a, b) => a + b)}');
}

void main() {
  var [m, n] = stdin.readLineSync()!.split(' ').map(int.parse).toList();
  while (m > 0 && n > 0) {
    formattedPrint(m, n);
    [m, n] = stdin.readLineSync()!.split(' ').map(int.parse).toList();
  }
}
